overlord/devicestate/handlers_install.go

```diff
@@ -126,6 +126,40 @@ func writeLogs(rootdir string) error {
 	if err := gz.Flush(); err != nil {
 		return fmt.Errorf("cannot flush compressed log output: %v", err)
 	}
+
+	return nil
+}
+
+func writeTimings(rootdir string, st *state.State) error {
+	logPath := filepath.Join(rootdir, "var/log/install-timings.txt.gz")
+	if err := os.MkdirAll(filepath.Dir(logPath), 0755); err != nil {
+		return err
+	}
+
+	f, err := os.Create(logPath)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	gz := gzip.NewWriter(f)
+	defer gz.Close()
+
```

Comment:
as discussed we might want to prefix the timings with the output of "snap changes" itself